Mailbox.java

```diff
@@ -0,0 +1,22 @@
+import java.util.ArrayList;
+
+public class Mailbox {
+	
+	private ArrayList<Message> messageQueue = new ArrayList();
```

Comment:
I think that Dr. Li wants us to use JavaDoc to document our code. Could you add more comments?